### PR TITLE
Branch1

### PR DESCRIPTION
**Pull request by moksh345**

### CODE REVIEW

File: applicatn.py [hunk 1]
```diff
@@ -42,6 +42,13 @@ def register():
         except Exception :
 	        return render_template("error.html", error = "Registration not succesfull")
 
+@app.route("/admin")
+
+def Member():
+      """List all users."""
+     
+      userlist = Users.query.all()
```
Comment (manasa133) [inline applicatn.py]: display the latest registration first

File: import.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+from flask_session import Session
+from flask_sqlalchemy import SQLAlchemy
+
+
+# Import table definitions.
+
+app1 = Flask(__name__)
+
+# Tell Flask what SQLAlchemy databas to use.
+app1.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
```
Comment (manasa133) [inline import.py]: you are already having create.py, that needs to do all these. why are you again writing in some files.

File: import.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+app1.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app1.app_context().push()
+# Link the Flask app1 with the database (no Flask app1 is actually being run yet).     
+
+db1 = SQLAlchemy()
+
+class Books(db1.Model):
```
Comment (manasa133) [inline import.py]: all the models must be in one file

File: import.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    __tablename__ = "BOOKS"
+    isbn = db1.Column(db1.String, nullable = False,primary_key=True)
+    tittle = db1.Column(db1.String, nullable = False)
+    author = db1.Column(db1.String, nullable = False)
+    year = db1.Column(db1.String, nullable= False)
+
+db1.init_app(app1)
```
Comment (manasa133) [inline import.py]: this should not be done as you are already doing in create.py

File: applicatn.py [hunk 5]
```diff
@@ -34,7 +34,7 @@ def register():
         name = request.form.get("name")
         password = request.form.get("password")
         userdata = Users(username=name, password=password, timestamp=str(datetime.now()))
```
Comment (manasa133) [inline applicatn.py]: let the time get assigned in the class itself, instead of burdening the server

File: applicatn.py [hunk 6]
```diff
@@ -34,7 +34,7 @@ def register():
...
-        print(f"added the user {name}{password}******************************************************")
+        # print(f"added the user {name}{password}******************************************************")
         try:
             db.session.add(userdata)
```
Comment (manasa133) [inline applicatn.py:39]: how are you indicating registration is succesfull or not?

**Comment by manasa133** [comment]
work on the comments

**Comment by moksh345** [comment]
Hello Mam,
 They mentioned to write the table separately in assignment task on NGlms.
Hence I rewrote the code to create table.

**Comment by moksh345** [comment]
Hello Mam, it is mentioned in the registered.html page that registration is
successful. Tq

Regards,
Moksh

**Comment by moksh345** [comment]
Ok mam . Will work on it. Thank you.

File: applicatn.py (hunk 6 above)
Comment (moksh345) [reply applicatn.py:39]: Hello Mam, it is mentioned in the registered.html page that registration is
successful. Tq

File: applicatn.py (hunk 5 above)
Comment (moksh345) [reply applicatn.py:36]: Hello Mam,
 They mentioned to write the table separately in assignment task on NGlms.
Hence I rewrote the code to create table.

File: import.py (hunk 4 above)
Comment (moksh345) [reply import.py]: Hello Mam,
 They mentioned to write the table separately in assignment task on NGlms.
Hence I rewrote the code to create table.

File: import.py (hunk 3 above)
Comment (moksh345) [reply import.py]: Hello Mam,
 They mentioned to write the table separately in assignment task on NGlms.
Hence I rewrote the code to create table.

File: import.py (hunk 2 above)
Comment (moksh345) [reply import.py]: Hello Mam,
 They mentioned to write the table separately in assignment task on NGlms.
Hence I rewrote the code to create table.

File: applicatn.py (hunk 5 above)
Comment (moksh345) [reply applicatn.py]: done mam. please check. tq

File: applicatn.py (hunk 1 above)
Comment (moksh345) [reply applicatn.py]: done mam. pls check.tq

File: import.py (hunk 2 above)
Comment (manasa133) [reply import.py]: yes, get the instance of db from application.py and insert the books.

File: import.py (hunk 3 above)
Comment (manasa133) [reply import.py]: you need separate table, that can be written in the user table file itself

File: import.py (hunk 2 above)
Comment (moksh345) [reply import.py]: hello mam, made the necessary changes. pls check. tq

File: import.py (hunk 3 above)
Comment (moksh345) [reply import.py]: hello mam, made the necessary changes. pls check. tq

**Comment by manasa133** [comment]
Merge with master branch

**Comment by moksh345** [comment]
merging done mam. pls check. tq

**Pull request merged**